distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestCatalog.java

```diff
@@ -1926,6 +1926,14 @@ private void persistToWorkspace(int size) throws Exception {
 
     @Test
     public void testContentVersioning() throws Exception {
+
+        Configuration config = getAdminConfig().getConfiguration("ddf.catalog.history.Historian");
+        config.setBundleLocation(
+                "mvn:ddf.catalog.core/catalog-core-standardframework/2.10.0-SNAPSHOT");
```

Comment:
You can add it in if you want. Put `editConfigurationFilePut("etc/system.properties", "ddf.version", MavenUtils.asInProject().getVersion("ddf.test.itests", "test-itests-common"))` in `configureSystemSettings` in `AbstractIntegrationTest`